frontend/src/components/Form.js

```diff
@@ -56,6 +57,35 @@ export default function Form() {
   const [savePreference, setSavePreference] = React.useState(true);
   const [matchPreference, setMatchPreference] = React.useState('none');
 
+  /** Gather user inputs from form and send POST request to backend
+    * @param {Event} event
+   */
+  function handleFormSubmission(event) {
+    // TODO(#34): Validate input values (before allowing to submit)
+
+    // Override browser's default behvaior to execute POST request
+    event.preventDefault();
+
+    // Gather all form inputs into one object
+    const formDetails = {
+      timeAvailableUntil: timeAvailableUntil.getTime(),
```

Comment:
Is there any way to check that getTime() returns that time on the current day?

Reply:
Since users will be in different timezones and the times all get converted to UTC on the backend, I am not sure if converting the times to the current day would help us

Reply:
So what does getTime() return? I assumed getTime() would return a time without specifying the date. However, if getTime() supposedly returns a long (because timeAvailableUntil is a long in AddParticipantServlet), that means that time must be on some date. What is that date? Is it the current date in their timezone? That's what we would want. Milliseconds since epoch isn't dependent on timezone. So my question is: is there a way to check the value that getTime() returns?

Reply:
getTime() returns the number representing the milliseconds elapsed between 1 January 1970 00:00:00 UTC and the given date. What check do you want to do on that value?